### PR TITLE
Added border to the sign in form in the admin page for windows high contrast mode light

### PR DESCRIPTION
**Pull request by Lovelyfin00**
Fixes issue wagtail/wagtail#9350

image

### CODE REVIEW

File: client/scss/components/_workflow-timeline.scss [hunk 1]
```diff
@@ -41,6 +41,9 @@
     position: relative;
     flex-shrink: 0;
     background: theme('colors.white.DEFAULT');
+    @media (forced-colors: active) {
+      background: Canvas;
```
Comment (lb-) [inline client/scss/components/_workflow-timeline.scss:45]: If possible - can this change be split to a different PR - I know the change is small but it's best if one PR can solve one problem.

File: client/scss/layouts/_login.scss [hunk 2]
```diff
@@ -65,6 +65,11 @@
       max-width: calc(theme('spacing.32') * 3.5);
       padding: theme('spacing.12') theme('spacing.14');
     }
+
+    // border for windows high contrast mode, light
+    @media (forced-colors: active) {
+      border: 5px solid transparent;
```
Comment (lb-) [inline client/scss/layouts/_login.scss:71]: Just a thought. Should we use `outline` instead. This is not a focusable element so the outline won't be used here. Outline also helps for avoiding additional content size changes. 

https://developer.mozilla.org/en-US/docs/Web/CSS/outline

**Comment by Lovelyfin00** [comment]
@lb- 
I've checked the link you sent and outline is basically for focusable elements like input element. Since I can't use outline here, should i reduce the size of the border or is the style transparent not good?

**Comment by Lovelyfin00** [comment]
But I tested it with outline solid and this is how it looks in light mode

image

Then this is how it looks in dark mode

image

**Review by lb- — CHANGES_REQUESTED** [review]
@Lovelyfin00  I think there has been some confusion with our chat on slack.

I recommended that we put the transparent background on the `content-wrapper` but not using the forced colours at all.

This way we get a simpler solution and no risk of 'shifting' of content size due to the border not existing on the non-forced colors mode variant.

Are you able to remove the forced colours mode on line 70 to 72 and then add.

```
      border: 5px solid transparent;
```

in place of line 54 - then see if it works in normal, plus light & dark high contrast mode. My understanding is that Windows will read there is a border and change the colour to the suitable colour based on the mode.

If this does not work we can discuss a different solution.

Hope that makes sense.

**Comment by lb-** [comment]
I think we can resolve the icon in a different PR - feel free to raise a new issue for this problem though with some screenshots.

**Comment by Lovelyfin00** [comment]
@lb- 
I uploaded these screenshots before I chatted you up.
I've made the necessary changes as we discussed.

Screenshot for light mode 
image


Screenshot for dark mode

image

**Comment by lb-** [comment]
Awesome - will aim to do another review if no one else gets to it before me

**Review by lb- — APPROVED** [review]
Thanks @Lovelyfin00  a few rounds but we got there - thanks for sticking at it.

This is a much more simpler solution and works great.

### Screenshots
Screen Shot 2022-10-20 at 10 49 11 pm
Screen Shot 2022-10-20 at 10 49 00 pm

**Pull request merged**